test/e2e-helm/e2e_helm_suite_test.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e_helm_test
+
+import (
+	"path/filepath"
+	"strings"
+	"testing"
+	"time"
+
+	. "github.com/onsi/ginkgo" //nolint:golint
+	. "github.com/onsi/gomega" //nolint:golint
+
+	testutils "github.com/operator-framework/operator-sdk/test/internal"
+	"github.com/operator-framework/operator-sdk/version"
+)
+
+// TestE2EHelm ensures the Helm projects built with the SDK tool by using its binary.
+func TestE2EHelm(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping Operator SDK E2E Helm Suite testing in short mode")
+	}
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "E2EHelm Suite")
+}
+
+var (
+	tc testutils.TestContext
+	// isPromethuesManagedBySuite is true when the suite tests is installing/uninstalling the Prometheus
+	isPromethuesManagedBySuite = true
+	// isOLMManagedBySuite is true when the suite tests is installing/uninstalling the OLM
+	isOLMManagedBySuite = true
+	// Kubectx staore the k8s context from where the tests are running
+	Kubectx string
```

Comment:
Typo and probably doesn't need to be exported.
```suggestion
	// kubectx stores the k8s context from where the tests are running
	kubectx string
```